crates/driver/src/infra/config/cli.rs

```diff
@@ -143,6 +181,77 @@ impl std::fmt::Display for ContractAddresses {
     }
 }
 
-pub fn hex_address(value: &str) -> eth::H160 {
-    eth::H160::from_str(value).unwrap()
+/// Parameters related to settlement submission.
+#[derive(Debug, clap::Parser)]
+pub struct Submission {
+    /// How to calculate gas prices. Multiple approaches are used in sequence if
+    /// the previous one fails.
+    /// eth-gas-station: supports mainnet.
+    /// gas-now: supports mainnet.
+    /// gnosis-safe: supports mainnet and goerli.
+    /// web3: supports every network.
+    /// native: supports every network.
+    #[clap(
+        long,
+        env,
+        default_value = "web3",
+        value_enum,
+        use_value_delimiter = true
+    )]
+    pub submission_gas_price_calculation: Vec<GasPriceCalculation>,
+
+    /// Additional tip in percentage of max_fee_per_gas we are willing to give
+    /// to miners above regular gas price estimation. Expects a floating point
+    /// value between 0 and 1.
+    #[clap(long, env, default_value = "0.05")]
+    pub submission_additional_tip_percentage: f64,
+
+    /// The maximum gas price in Gwei the solver is willing to pay in a
+    /// settlement.
+    #[clap(long, env, default_value = "1500")]
+    pub submission_gas_price_cap: f64,
+
+    /// The target confirmation time for settlement transactions used
+    /// to estimate gas price. Specified in seconds.
+    #[clap(long, env, default_value = "30")]
+    pub submission_target_confirm_time_secs: u64,
+
+    /// Amount of time to wait before retrying to submit the tx to
+    /// the ethereum network. Specified in seconds.
+    #[clap(long, env, default_value = "2")]
+    pub submission_retry_interval_secs: u64,
+
+    /// Don't submit transactions with high revert risk (i.e. transactions that
+    /// interact with on-chain AMMs) to the public mempool. This can be
+    /// enabled to avoid MEV when private transaction submission strategies
+    /// are available.
+    #[clap(long, env)]
+    pub submission_disable_high_risk_public_mempool_transactions: bool,
+
+    /// The maximum time to spend trying to settle a transaction
+    /// through the Ethereum network before going back to solving. Specified in
+    /// seconds.
```

Comment:
> I think how the new driver is currently implemented already supports solving a new auction while it still tries to bring the other one on-chain

I think there will be dragons if we try to `/execute` multiple settlements while a previous one is still pending.

Might actually make sense to allow `/solve`-ing, but reject `/execute` requests in case a previous settlement transaction is still "in flight".